lib/iris/fileformats/netcdf.py

```diff
@@ -2264,7 +2264,13 @@ def is_valid_packspec(p):
                 warnings.warn(msg)
 
         # Add conventions attribute.
-        sman.update_global_attributes(Conventions=conventions)
+        if (cube.attributes.has_key('Conventions')):
+            msg = 'Keeping conventions from original netcdf file'
+            warnings.warn(msg)
+            sman.update_global_attributes(Conventions=cube.attributes.get('Conventions'))
+        else:
+            sman.update_global_attributes(Conventions=conventions)
```

Comment:
@morwenna-01 You seem to have backed out my change here. We need to move your override to [here](https://github.com/morwenna-01/iris/pull/1/commits/dbda1d06c3a5d2cbd9bc0d63fc5ca35f6674c9a2#diff-e055d3568b9b3f9cfe397cec7e016700R2251) otherwise the preceding patching breaks. Plus the usage of `has_key` is not Python3 friendly.